xga/utils.py

```diff
@@ -206,7 +206,10 @@ def build_observation_census(tel: str) -> Tuple[pd.DataFrame, pd.DataFrame]:
 
                         # We check that the filter value isn't in the list of unacceptable filters for the
                         #  current telescope
```

Comment:
You should add something to this comment saying that not all telescopes _have_ a filter header to check - which is why you did this if statement